code-challenges.js

```diff
@@ -9,67 +9,248 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
+//setting uyp fonction that takes temp var as argument 
+//returns string of "${temp var} is below boiling point"
+//or returns string of "${temp var} is above boiling point"
+//or returns string of "${temp var} is at boiling point"
+
+
+
+
 var temp1 = 85
 var temp2 = 350
 var temp3 = 212
 
 
+const tempChecker = (temp) =>{
+    if (temp <= 211)
+    return `${temp} is below boiling point`
+    else if (temp >= 213)
+    return `${temp} is above boiling point`
+    else if (temp === 212)
+    return `${temp} is at boiling point`
+    else 
+    return "try again"
+
+}
+
+console.log(tempChecker(temp1))
+console.log(tempChecker(temp2))
+console.log(tempChecker(temp3))
+
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
+
+//setup function that takes an array as the argument 
+//using a for loop multiply each number by 5
+
+
 var myForNumbers1 = [3, 7, 0, 6, -9]
 var myForNumbers2 = [2, 5, -8, 20, 32]
 
+const multiplyByFive = (array) => {
+let multiplied = [];
+for (let i =0  ; i < array.length ; i++){
+    multiplied.push (array[i] * 5);
```

Comment:
```suggestion
    multiplied.push(array[i] * 5);
```
for executing functions no space is needed between parentheses